### PR TITLE
News article added.

### PR DESCRIPTION
**Pull request by kumardennis**

### CODE REVIEW

**Comment by edanchenkov** [comment]
Why there are 4 new images added?

File: src/pages/news/first-harmonized-data-model-published-article/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+path: '/news/first-harmonized-data-model-published-article'
+date: '2019-10-04T13:45:33.962Z'
+title: 'First harmonized data model published'
+image: './../../static/harmonized-data-model.png'
+pic: 'harmonized-data-model-square.png'
+pictext: ''
```
Comment (edanchenkov) [inline src/pages/news/first-harmonized-data-model-published-article/index.md:7]: Is this becoming "alt" text in html? If so then it is better to have it for SEO and accessibility purposes.

File: src/pages/news/first-harmonized-data-model-published-article/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+type: 'news'
+subtype: 'article'
+tags: ['First Harmonized', 'Data Model']
```
Comment (edanchenkov) [inline src/pages/news/first-harmonized-data-model-published-article/index.md:10]: This could be good to verify with marketing people

**Comment by kumardennis** [comment]
I wasnt sure if the images should be only in static folder, or should also in the main folder images where all images are collective? Could it be clarified? @edanchenkov

File: src/pages/news/first-harmonized-data-model-published-article/index.md (hunk 1 above)
Comment (kumardennis) [reply src/pages/news/first-harmonized-data-model-published-article/index.md:7]: havent seen pictxt being filled in other news items. And it turns to be into  <p> as caption.

Comment (edanchenkov) [reply src/pages/news/first-harmonized-data-model-published-article/index.md:7]: Ok then we might take care of it at some point later. It is up to you now.

**Comment by edanchenkov** [comment]
@kumardennis Well, could you test it and see. As far as I remember they can be in any place as long as path to them is correct. Build process will sort it out for you.

**Review by edanchenkov — APPROVED** [review]

**Review by victorv-sbp — APPROVED** [review]

**Pull request merged**